src/v/rpc/transport.cc

```diff
@@ -219,22 +219,31 @@ void transport::dispatch_send() {
                          || _requests_queue.begin()->first
                               > (_last_seq + sequence_t(1));
               },
-              [this] {
```

Comment:
> rpc/transport: coroutinize part of dispatch_send

this commit message doesn't seem to match up with what is happening in the commit.

Reply:
Changed the commit message